### DISCUSSION TITLE
Can let gleitzKitUrl baseUrl in external can replace, https://gleitz.github.io/midi-js-soundfonts/

### DISCUSSION THREAD

**Discussion by AlanPaine** [danigb/smplr#12] [Q&A]
Can let gleitzKitUrl baseUrl in external can replace, https://gleitz.github.io/midi-js-soundfonts/

**Comment by danigb** [comment]
You can use any valid url. See documentation

**Reply by AlanPaine** [reply]
For example, I want to use the URl address in China, because it is slow to visit foreign countries in China

**Reply by danigb** [reply]
You need to setup a server in China to store the Soundfonts and access that server

**Reply by AlanPaine** [reply]
I think by setting it like this, I don't have to replace every single one, so I just set the whole thing

**Reply by AlanPaine** [reply]
> 您需要在中国设置一个服务器来存储 Soundfonts 并访问该服务器

Yes, I've already set up a server

**Reply by danigb** [reply]
This is out of the scope of the library. Please search for help about servers in another place.

**Reply by AlanPaine** [reply]
You don't understand what I'm saying, I mean let's repository gleitzKitUrl function method is the return of ` kit} {https://gleitz.github.io/midi-js-soundfonts/$/ ${name} - ${format}. Js `;

} I want to control the request address here by outsourcing
